.github/workflows/qa-deployment.yml

```diff
@@ -0,0 +1,57 @@
+name: Deploy to QA environment
+
+on:
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  deploy-to-qa:
+    if: github.repository == 'AOT-Technologies/forms-flow-ai'
+    runs-on: ubuntu-latest
+    environment: 'qa'
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - name: Set up AWS CLI
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ca-central-1  # Change this to your desired region
+      - name: Install kubectl
+        run: |
+          curl -LO "https://dl.k8s.io/release/$(curl -Ls https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+          chmod +x kubectl
+          sudo mv kubectl /usr/local/bin/
+      - name: Update Kube config
+        run: aws eks update-kubeconfig --region ca-central-1 --name forms-flow-eks-staging
+      
+      - name: Deploy to QA
+        run: |
+          kubectl -n qa patch deployment forms-flow-api -p '{"spec":{"template":{"spec":{"containers":[{"name":"forms-flow-api","image":"docker.io/formsflow/forms-flow-webapi:${{ env.VERSION }}"}]}}}}'
+          kubectl -n qa patch deployment forms-flow-bpm -p '{"spec":{"template":{"spec":{"containers":[{"name":"forms-flow-bpm","image":"docker.io/formsflow/forms-flow-bpm:${{ env.VERSION }}"}]}}}}'
+          kubectl -n qa patch deployment forms-flow-documents-api -p '{"spec":{"template":{"spec":{"containers":[{"name":"forms-flow-documents-api","image":"docker.io/formsflow/forms-flow-documents-api:${{ env.VERSION }}"}]}}}}'
+          kubectl -n qa patch deployment forms-flow-web -p '{"spec":{"template":{"spec":{"containers":[{"name":"forms-flow-web","image":"docker.io/formsflow/forms-flow-web:${{ env.VERSION }}"}]}}}}'
+      
+      - name: Deploy to QA ee
```

Comment:
this one confused to me. ee deployment from open source?

Reply:
Yes. For QA, they would be deploying as and when feature is ready as per dev communication. And they would need to test it in all QA environments; so that's why added all env actions as part of a single action; and they wouldn't need to go to multiple places to do this and at the same time it's fully on-demand and not tied to any folder changes.